src/components/Template/Template.tsx

```diff
@@ -90,7 +90,7 @@ const TemplateHeader = ({
   children,
   renderHeaderElement = true,
 }: React.PropsWithChildren<TemplateHeaderProps>) => {
-  const styles = useStyleConfig("TemplateHeader", {});
+  const styles = useStyleConfig("TemplateBreakout", {});
```

Comment:
Did you find commonalities between `TemplateHeader` and `TemplateBreakout` and determined that one could be removed?

Reply:
Correct. The only difference was the name of the object.